tests/console/verify_systemd_timesync.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify that systemd timer is used for time synchronization.
+#          - Check if configuration file exists and contains expected values.
+#          - Check the message logs for "One time synchronization" occurance.
+# Maintainer: Sofia Syrianidou <ssyrianidou@suse.com>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+sub run {
+    my $test_data = get_test_suite_data();
+    select_console 'root-console';
+
+    record_info("Check configuration", "Check if file /etc/systemd/system/yast-timesync.timer exists and has expected configuration.");
+    assert_script_run('ls /etc/systemd/system/yast-timesync.timer');
+    my $conf_file = script_output("cat /etc/systemd/system/yast-timesync.timer");
```

Comment:
`/etc/systemd/system/yast-timesync.timer` looks like a good candidate to be a variable as it is used widely. i would suggest to introduce a variable instead. how does it sound?

Reply:
I am not sure to be honest. Wouldn't hurt to replace it by variable but I don't see the benefit or the logic behind it because we don't expect it to change. You are proposing this for better code legibility?

Reply:
as general principle you store it in a variable and
- any change happens in one place. This comes with the obvious benefits. for instance it can avoid mistypings when refactoring etc
- makes the code more readable usually using a good name  
- there are other concepts also invoking CPU and memory (out of scope here and do not make any difference)

Reply:
Hmm. OK, I will keep that in mind for future tests, but here as I don't expect this to change as it is used two times, I will leave it as it is.